common/transport/amqp/package.json

```diff
@@ -7,24 +7,25 @@
   "main": "main.js",
   "typings": "index.d.ts",
   "dependencies": {
-    "amqp10": "3.5.0",
+    "@types/async": "^2.0.42",
+    "amqp10": "^3.5.3",
```

Comment:
>^3.5.3 [](start = 15, length = 6)

We should talk about use of the carat and tilde operator in package.json.  For stability, I wonder if we should lock to specific versions.  (Yes, I know how semver and these operators _should_ work, but real life isn't always as clean, especially with packages that have limited support)

Reply:
yes, we should fixate on a specific version. also, i have ideas how to integrate dependencies updates and vulnerability scanning in our CI

---
In reply to: [139485216](https://github.com/Azure/azure-iot-sdk-node/pull/83#discussion_r139485216) [](ancestors = 139485216)